tests/torch_api/data_parallel/test_c10d_common.py

```diff
@@ -44,6 +48,7 @@
 )
 
 import bagua.torch_api.data_parallel.functional as bagua_dist
+from bagua.torch_api.contrib.sync_batchnorm import _SYNC_BN_V5, _SYNC_BN_V6, _SYNC_BN_V7
 
```

Comment:
I don't think we should couple this test with sync bn together.

Reply:
@wangraying This approach aims to make the test cases compatible with multiple versions of PyTorch.

Do you have any other suggestions for addressing the compatibility issues with multiple versions of PyTorch?

Reply:
Yeah, I agree using `LooseVersion**` as the condition, but it's kind of confusing to import the condition from sync bn.